doc/common/references.md

```diff
@@ -37,13 +38,64 @@
+| | | "Multus". Available at https://github.com/intel/multus-cni. |
+| | | "Node Feature Discovery (NFD)". Available at https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html. |
+| | | "Open Container Initiative (OCI)". Available at https://github.com/opencontainers/runtime-spec. |
+
 
 <a name="1.3"></a>
-## RA References
+## Network related References
 
-<a name="1.4"></a>
-## RI References
+|Ref	     | Doc Number	            |Title          |
+|--------  |------------------------|---------------|
+| | IETF-RFC 2544 | "Benchmarking Methodology for Network Interconnect Devices".  Available at https://datatracker.ietf.org/doc/html/rfc2544. |
+| | ITU-T Y.3500  | "Information Technology – Cloud computing – Overview and vocabulary"; SERIES Y: Global Information Infrastructure, Internet Protocol Aspects and Next-Generation Networks: Cloud Computing. Available at https://www.itu.int/rec/T-REC-Y.3500/recommendation.asp?lang=en&parent=T-REC-Y.3500-201408-I. |
+| | | "P4 Language Consortium" (Packet Processing Programming). Available at https://p4.org/.|
 
 <a name="1.4"></a>
-## RC References
+## O-RAN, 5G and Miscellaenous References
+
+|Ref	     | Doc Number	            |Title          |
+|----------|------------------------|---------------|
+| | ITU-T IMT-2020 | "International Mobile Telecommunications-2020 (IMT-2020) Standard for 5G networks". Available at https://www.itu.int/pub/T-TUT-IMT. |
+| | O-RAN.WG6.AAL-GAnP-v01.00 | "O-RAN Acceleration Abstraction Layer General Aspects an Principles 1.0", November 2020; O-RAN.WG6.AAL-GAnP-v01.00. Available at https://www.o-ran.org/specifications. |
+| | ETSI TS 123 501 V16.6.0 | "System architecture for the 5G System (5GS)". ETSI TS 123 501 V16.6.0 (2020-10) (3GPP TS 23.501 version 16.6.0 Release 16). Available at https://www.etsi.org/deliver/etsi_ts/123500_123599/123501/16.06.00_60/ts_123501v160600p.pdf. |
+
+<a name="1.5"></a>
+## Security Related References
+
+|Ref	     | Doc Number	            |Title          |
+|----------|------------------------|---------------|
+| | | CSA's Information Security Management through Reflexive Security. Available at https://cloudsecurityalliance.org/artifacts/information-security-management-through-reflexive-security/. |
+| | | ETSI Industry Specification Group Network Functions Virtualisation (ISG NFV).  Available at https://www.etsi.org/committee/1427-nfv. |
+| | | ETSI Cyber Security Technical Committee (TC CYBER).  Available at https://www.etsi.org/committee/cyber
+| | ETSI GS NFV-SEC 014 V3.1.1 (2018-04) |  Network Functions Virtualisation (NFV) Release 3; "NFV Security; Security Specification for MANO Components and Reference points";  ETSI GS NFV-SEC 014 V3.1.1 (2018-04). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/014/03.01.01_60/gs_NFV-SEC014v030101p.pdf. |
```

Comment:
```suggestion
| | ETSI GS NFV-SEC 014 V3.1.1 |  Network Functions Virtualisation (NFV) Release 3; "NFV Security; Security Specification for MANO Components and Reference points";  ETSI GS NFV-SEC 014 V3.1.1 (2018-04). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/014/03.01.01_60/gs_NFV-SEC014v030101p.pdf. |
```
Suggest to omit the date in the Doc Number column.

Reply:
@ulikleber The references in ETSI documents includes the dates